Tools/scripts/build_options.py

```diff
@@ -43,6 +43,7 @@ def config_option(self):
     Feature('AHRS', 'EKF3_EXTNAV', 'EK3_FEATURE_EXTERNAL_NAV', 'Enable External navigation for EKF3', 0, 'EKF3'),
     Feature('AHRS', 'EKF3_WINDEST', 'EK3_FEATURE_DRAG_FUSION', 'Enable Wind estimation for EKF3', 0, 'EKF3'),
     Feature('AHRS', 'EKF3_OPTFLOW', 'EK3_FEATURE_OPTFLOW_FUSION', 'Enable OpticalFlow fusion for EKF3', 0, 'EKF3,OPTICALFLOW'),
+    Feature('AHRS', 'EKF3_OPTFLOW_SRTM', 'EK3_FEATURE_OPTFLOW_SRTM', 'Enable OpticalFlow using SRTM for EKF3', 0, 'EKF3,OPTICALFLOW'),  # noqa: E501
```

Comment:
```suggestion
    Feature('AHRS', 'EKF3_OPTFLOW_SRTM', 'EK3_FEATURE_OPTFLOW_SRTM', 'Enable OpticalFlow using SRTM for EKF3', 0, 'EKF3_OPTFLOW'),  # noqa: E501
```

```
 ./Tools/autotest/test_build_options.py --define-match-glob EKF3_OPTFLOW_SRTM
```

Reply:
ok, although I don't see why we have a build option for, "EK3_FEATURE_OPTFLOW_FUSION".  If the user has an optical flow sensor attached why would they not want EKF3 fusion?  I think we should remove EK3_FEATURE_OPTFLOW_FUSION and instead simply use OPTICALFLOW